integration-tests/quarkus-sample/src/main/java/com/azure/microprofile/it/quarkus/ConfigResource.java

```diff
@@ -0,0 +1,51 @@
+package com.azure.microprofile.it.quarkus;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.StreamSupport;
+
+@Path("/config")
+public class ConfigResource {
+
+    @Inject
+    private Config config;
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/value/{name}")
+    public String getConfigValue(@PathParam("name") String name) {
+        return config.getConfigValue(name).getValue();
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/propertyNames")
+    public Set<String> getConfigPropertyNames() {
+        ConfigSource configSource = getConfigSource("AzureKeyVaultConfigSource");
```

Comment:
Is it possible to use a const or something like AzureKeyVaultConfigSource.class.getSimpleName()?

Reply:
Good point, addressed in [87565b6](https://github.com/Azure/azure-microprofile/pull/7/commits/87565b6543010fe01d9d4b8acb5d5a5a66b39f8b).